utils/stubs.py

```diff
@@ -0,0 +1,100 @@
+# coding: Latin-1
+# Copyright © 2018 The Things Network
+# Use of this source code is governed by the
+# MIT license that can be found in the LICENSE file.
+
+
+from jose import jwt, jws
+from .read_key import read_key
+import math
+import binascii
+import time
+import os
+
+MODERN_CIPHER = ("ECDHE-ECDSA-AES256-GCM-SHA384:"
+                 "ECDHE-RSA-AES256-GCM-SHA384:"
+                 "ECDHE-ECDSA-CHACHA20-POLY1305:"
+                 "ECDHE-RSA-CHACHA20-POLY1305:"
+                 "ECDHE-ECDSA-AES128-GCM-SHA256:"
+                 "ECDHE-RSA-AES128-GCM-SHA256:"
+                 "ECDHE-ECDSA-AES256-SHA384:"
+                 "ECDHE-RSA-AES256-SHA384:"
+                 "ECDHE-ECDSA-AES128-SHA256:"
+                 "ECDHE-RSA-AES128-SHA256")
+
+uplink = {
```

Comment:
Are `uplink`, `claims`, `appClaims` still used at all?

Reply:
uplink is still used in the test for the MQTT client but claims and appClaims are not used anymore, I'm removing them